lib/node/node.js

```diff
@@ -187,6 +188,14 @@ class Node extends EventEmitter {
       this.logger.warning('Warning: worker pool is disabled.');
       this.logger.warning('Verification will be slow.');
     }
+
+    // native: 0 = js, 1 = node, 2 = bindings
+    if (sha256.native !== 2 &&
+        process.env.NODE_BACKEND !== 'js' &&
```

Comment:
The boolean logic here reads a bit difficult.

If its not natively built and the node backend isn't explicitly js and the node backend isn't explicitly node then throw an error. Converting to an equivalent logical statement using [De Morgan's Law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws)  that reads better in my head: if its not natively built and not (the node backend is js or the node backend is node), then throw an error.

This will need to be updated if the wasm backend ever lands.

Writing it out helped me grok it better. My only nit is that I think longer boolean statements read better when the boolean operator is in the front, things are symmetrical that way. No biggie if you don't agree :), ie 

```javascript
if (sha256.native !== 2
    && process.env.NODE_BACKEND !== 'js'
    && process.env.NODE_BACKEND !== 'node') {
  throw new Error()
}
```